cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -76,4 +76,33 @@ or provide Twitter API credentials::
       queue = new LinkedBlockingQueue<Tweet>(10000);
       collector = new CollectingThread();
       collector.start();
-    }
\ No newline at end of file
+    }
+
+Scoped Runtime Arguments for Workflow
+=====================================
+When a Workflow is configured, you may want to pass specific runtime arguments to the different programs
+and datasets used inside the Workflow. To achieve this, you can prefix the runtime arguments with a ``<scope>.``.
```

Comment:
Remove the extra period inside the literal -- I think it will be too confusing having both,